### ISSUE TITLE
bh export causes the server to hang

### ISSUE THREAD

**Issue by ghost** [grandwazir/BanHammer#115]
as the title says, 15k bans cause bh export to hang the server

**Comment by grandwazir** [comment]
Yes it might do. The bans have to be exported on the main thread because of the way the server is organised. I can add a feature to schedule exporting the list over a period of time which might help the situation.

**Comment by TheWhiiteNinja** [comment]
I want to ask what plugins is using banhammer ?

**Comment by TheWhiiteNinja** [comment]
And can you make the plugin for 1.8.x and for bungee cord
